tests/lcs/core/files/bom.livecodescript

```diff
@@ -0,0 +1,38 @@
+﻿script "CoreFilesBOM"
+/*
+Copyright (C) 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+constant kStackName = "utf8bom"
+constant kTestFile = "utf8bom.livecodescript"
+
+on TestLoadBomScriptOnlyStack
+   local tUrlBinfilePath
+   
+   put quote & "binfile:" & testFilesFolder() & slash & \
+      kTestFile & quote into tUrlBinfilePath
+   
+   open stack url tUrlBinfilePath
+   
+   // Technically this is testing the opening of a stack that
+   // is coming from a chunk.  Most likely source would be URL
+   // "binfile:" and "http(s):", but others could be possible.
```

Comment:
Yes, people have been known to stuff stacks into custom properties. It would be good to add a test without the BOM here too. BTW GitHub diff is not showing me the BOM on the file. I can't recall if it normally does or not.

Reply:
I could change over to writing a file out and then opening it if this method isn't desired.  I did add tests that use a constructed variable for the stack (with and without a BOM).